target/scripts/build/packages.sh

```diff
@@ -0,0 +1,101 @@
+#! /bin/bash
+
+set -eE -u -o pipefail
```

Comment:
I'm not too good with memory, but I'm probably the odd one out here with active maintainers currently. Is there any value including a brief comment that describes what is accomplished? (Optional)

Reply:
I can provide one :)